car_control.py

```diff
@@ -1,137 +1,128 @@
+        "\nThrottle Servo Max: ",self.THROTTLE_SERVO_MAX,"\nSteering Servo Min: ",self.STEERING_SERVO_MIN,"\nSteering Servo Max: ",self.STEERING_SERVO_MAX, file=self.vari_file, flush=True)
     
     def figure_out_my_steering(self, line):
         [vx,vy,x,y] = line
-        cx_middle = x - (FRAME_WIDTH / 2)
-        cx_normal = cx_middle / (FRAME_WIDTH / 2)
+        cx_middle = x - (self.FRAME_WIDTH / 2)
+        cx_normal = cx_middle / (self.FRAME_WIDTH / 2)
     
         if self.old_cx_normal != None:
-            self.old_cx_normal = (cx_normal * MIXING_RATE) + \
-                (self.old_cx_normal * (1.0 - MIXING_RATE))
+            self.old_cx_normal = (cx_normal * self.MIXING_RATE) + \
+                (self.old_cx_normal * (1.0 - self.MIXING_RATE))
         else: 
             self.old_cx_normal = cx_normal
         return self.old_cx_normal
 
     def figure_out_my_throttle(self, steering): # steering -> [0:180]
-        # Solve: THROTTLE_CUT_OFF_RATE = pow(sin(90 +/- THROTTLE_CUT_OFF_ANGLE), x) for x...
-        #        -> sin(90 +/- THROTTLE_CUT_OFF_ANGLE) = cos(THROTTLE_CUT_OFF_ANGLE)
-        t_power = math.log(THROTTLE_CUT_OFF_RATE) / math.log(math.cos(math.radians(THROTTLE_CUT_OFF_ANGLE)))
+        # Solve: self.THROTTLE_CUT_OFF_RATE = pow(sin(90 +/- self.THROTTLE_CUT_OFF_ANGLE), x) for x...
+        #        -> sin(90 +/- self.THROTTLE_CUT_OFF_ANGLE) = cos(self.THROTTLE_CUT_OFF_ANGLE)
+        t_power = math.log(self.THROTTLE_CUT_OFF_RATE) / math.log(math.cos(math.radians(self.THROTTLE_CUT_OFF_ANGLE)))
     
         # pow(sin()) of the steering angle is only non-zero when driving straight... e.g. steering ~= 90
         t_result = math.pow(math.sin(math.radians(max(min(steering, 179.99), 0.0))), t_power)
-        return (t_result * THROTTLE_GAIN) + THROTTLE_OFFSET
+        return (t_result * self.THROTTLE_GAIN) + self.THROTTLE_OFFSET
     
     #
     # Servo Control Code
     #
     
-    # throttle [0:100] (101 values) -> [THROTTLE_SERVO_MIN, THROTTLE_SERVO_MAX]
-    # steering [0:180] (181 values) -> [STEERING_SERVO_MIN, STEERING_SERVO_MAX]
-    def set_servos(self, throttle, steering):
-        throttle = THROTTLE_SERVO_MIN + ((throttle/100) * (THROTTLE_SERVO_MAX - THROTTLE_SERVO_MIN))
-        steering = STEERING_SERVO_MIN + ((steering/180) * (STEERING_SERVO_MAX - STEERING_SERVO_MIN))
-        servo3.set(throttle)
-        servo1.set(steering)
+    # throttle [0:100] (101 values) -> [self.THROTTLE_SERVO_MIN, self.THROTTLE_SERVO_MAX]
+    # steering [0:180] (181 values) -> [self.STEERING_SERVO_MIN, self.STEERING_SERVO_MAX]
+    # def set_servos(self, throttle, steering):
+        # throttle = self.THROTTLE_SERVO_MIN + ((throttle/100) * (self.THROTTLE_SERVO_MAX - self.THROTTLE_SERVO_MIN))
+        # steering = self.STEERING_SERVO_MIN + ((steering/180) * (self.STEERING_SERVO_MAX - self.STEERING_SERVO_MIN))
+        # servo3.set(throttle)
+        # servo1.set(steering)
```

Comment:
Let's make this some kind of global configuration. I think the function definition should not be a problem when there is no hardware, just the references to servo1 and servo3.